### DISCUSSION TITLE
Why don't use pattern scan / AOB scan?

### DISCUSSION THREAD

**Discussion by HundSimon** [HunterPie/HunterPie#558] [Ideas]
Every time capcom updates monster hunter world it needs time to do remapping and publish a new version. So why don't you use something like pattern scan? It can handle code updates that don't associated with basic codes. This is achievable technically.
I saw some works ( https://github.com/HunterPie/HunterPie/pull/40 ) on it but i don't really understand why not to use pattern scan to get all data needed, rather than base addresses only.

**Comment by haato3o** [comment]
I do use a pattern scanner, so it's really not a big deal for me. Not embedding the scanner in HunterPie is a decision I made because this way I can ensure all addresses are working properly after an update. Also, even pattern scanning is not future proof as I had some signatures break before.

**Reply by HundSimon** [reply]
Thanks for answering my question!
